src/openqaoa-core/openqaoa/algorithms/baseworkflow.py

```diff
@@ -271,25 +273,46 @@ def set_error_mitigation_properties(self, **kwargs):
         Parameters
         ----------
             error_mitigation_technique: str
-                The specific technique used to mitigate the errors. Only a simple state preparation and measurement twirling with bitflip averages, under the name "spam_twirling" is currently supported.
+                The specific technique used to mitigate the errors. Currently, the availables techniques are:
+                    *   A simple state preparation and measurement twirling with bitflip averages, under the name "spam_twirling".
+                    *   Zero Noise Extrapolation (ZNE), integrated from Mitiq framework.
             n_batches: int
                 The number of batches specifies the different negating schedules at random. Total number of shots is distributed accordingly.
             calibration_data_location: str
-                The location of the json file containing calibration data. For spam twirling this is the measurement outcomes of an empty circuit under the bit-flip averaging.
-
+                The location of the json file containing calibration data.
+                    *   For spam twirling this is the measurement outcomes of an empty circuit under the bit-flip averaging.
+                    *   For Zero Noise Extrapolation (ZNE) these are the factory, the scaling, the seed(only for fold_gates_at_random scaling)
+                        and the scale factors.
         """
+
+        # validate a supported error mitigation technique
+        if kwargs["error_mitigation_technique"].lower() in self.available_error_mitigation_techniques:
+            pass
+        else:
+            raise ValueError(
+                    f"Specified error mitigation technique is not supported"
+                )
+
+        # get the ErrorMitigationProperty structure to validate
+        error_mitigation_technique = kwargs["error_mitigation_technique"].lower()
+        if error_mitigation_technique == 'mitiq_zne':
+            self.error_mitigation_properties = Mitiq_Zne_ErrorMitigationProperties()
+            error_mitigation_properties = Mitiq_Zne_ErrorMitigationProperties
```

Comment:
Removed line `self.error_mitigation_properties = Mitiq_Zne_ErrorMitigationProperties()`
and also line `self.error_mitigation_properties = Spam_Twirling_ErrorMitigationProperties()`

The following for loop only acts on the temporary variable error_mitigation_properties, and in case it succeeds, the corresponding object is created and set into the Workflow internal attribute.

Reply:
I had to add the following fix: `self.error_mitigation_properties = error_mitigation_properties()` after the if-else block. This is necessary: python can only check if an attribute belongs to an instance, not to a class. We necessarily have to create an empty instance and then verify if the attribute belongs to that instance.

Reply:
Alright that makes sense, let's leave it as is